### ISSUE TITLE
Dialogue System Enhancement

### ISSUE THREAD

**Issue by Exocizcs** [Exocizcs/PixelFarm#3]
### Dialogue System Required

**Created new tile palettes**

Bunny Ground palette created for scene

**Created `Dialogue Folder`**

- Created `SimpleDialogue` script for handling dialogues with NPCs

**Issues**
- Boxes not able to appear even call DialogueBox at `Start` method

- Enhancement is appreciated if possible

_NEEDED A.S.A.P_

**Comment by Exocizcs** [comment]
If possible please do, thanks

**Comment by Exocizcs** [comment]
Regarding the dialogue system it was found that during the dialogue player still able to move and end the dialogue halfway through. Is there any way to stop that from happening? I wish to freeze player but not the animation

**Comment by choonhongyeoh0241** [comment]
Yes, it is possible but it would happen as in different issue. Could you open up a new issue for us to make changes?
